_posts/2024-03-29-geospatial-techno.md

```diff
@@ -0,0 +1,78 @@
+---
+author: GeoSpatial Techno
+layout: post
+title: How to style a layer using GeoServer and QGIS
+date: 2024-03-29
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### Using GeoServer and QGIS to style a layer
+In this session, we will explore the "Capabilities of SLD" and "How to create styles" to produce beautiful maps. If you want to access the complete tutorial, simply click on the [link](https://youtu.be/9AUgLFS9CCY)
+
+[![](https://img.youtube.com/vi/9AUgLFS9CCY/0.jpg)](https://www.youtube.com/watch?v=9AUgLFS9CCY)
+
+## Introduction
+Geospatial data has no intrinsic visual component and it must be styled to be visually represented on a map. GeoServer uses a markup language called Styled Layer Descriptor (SLD) to define styling rules for displaying data. SLD is an XML-based language that allows users and software to control the visual portrayal of geospatial data. This language ensures that clients and servers can both understand how to render the data visually.
+
+**Note.** In this blog post, we used GeoServer version 2.20.0.
```

Comment:
Why such an old version?  We try to encourage users to update regularly

Reply:
This blog post summarizes the tutorial video that goes with each post. I need to let users know upfront that the tutorial video is based on version 2.20.0. So, this warning will appear at the start of every blog post.

Reply:
That is a problem, a new blog should not talk about an unsupported GeoServer version.....

Reply:
While the work process generally remains similar across different versions of Geoserver, it is worth mentioning the specific version used in the videos to inform users accurately.
However, the primary source for information should always be the original Geoserver documentation.

Reply:
I think identifying the version used is a good idea. Using an unsupported version is a bad idea.

I think the options are either:

1. Use a currently supported version (as of time of posting)
2. Put this on a site that is not associated with the geoserver project.

We don't want to confuse the messaging.

Reply:
At the time of creating this video tutorial, I used the latest version of GeoServer. However, new versions have been developed and released since then.
In my opinion, these videos (featuring the old version) are helpful for teaching users the fundamentals of Geoserver through visual presentations. Additionally, we can create new videos that specifically cover the topics that have been modified in recent versions, distinguishing them from the older versions.

Reply:
A possible third option is to admit it was done using an outdated version, and clarify usage of unsupported versions is not recommended. Thinking out loud:

> This video was recorded some time ago on GeoServer 2.20.0, while the current supported series are, at the time of writing, 2.24.x and 2.25.x. Please do not use outdated versions of GeoServer, check the latest releases available at geoserver.org for production usage.

It's still not great, but at the very least it's not promoting usage of 2.20.0

Reply:
It's done